systemtest/src/main/java/io/strimzi/systemtest/resources/jaeger/SetupJaeger.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.resources.jaeger;
+
+import io.fabric8.kubernetes.api.model.networking.v1.NetworkPolicy;
+import io.fabric8.kubernetes.api.model.networking.v1.NetworkPolicyBuilder;
+import io.strimzi.systemtest.Constants;
+import io.strimzi.systemtest.resources.ResourceItem;
+import io.strimzi.systemtest.resources.ResourceManager;
+import io.strimzi.systemtest.utils.StUtils;
+import io.strimzi.systemtest.utils.kubeUtils.controllers.DeploymentUtils;
+import io.strimzi.test.TestUtils;
+import io.strimzi.test.k8s.KubeClusterResource;
+import io.strimzi.test.logs.CollectorElement;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Stack;
+
+import static io.strimzi.systemtest.Constants.JAEGER_DEPLOYMENT_POLL;
+import static io.strimzi.systemtest.Constants.JAEGER_DEPLOYMENT_TIMEOUT;
+import static io.strimzi.systemtest.tracing.TracingConstants.CERT_MANAGER_CA_INJECTOR_DEPLOYMENT;
+import static io.strimzi.systemtest.tracing.TracingConstants.CERT_MANAGER_DEPLOYMENT;
+import static io.strimzi.systemtest.tracing.TracingConstants.CERT_MANAGER_NAMESPACE;
+import static io.strimzi.systemtest.tracing.TracingConstants.CERT_MANAGER_WEBHOOK_DEPLOYMENT;
+import static io.strimzi.systemtest.tracing.TracingConstants.JAEGER_INSTANCE_NAME;
+import static io.strimzi.systemtest.tracing.TracingConstants.JAEGER_OPERATOR_DEPLOYMENT_NAME;
+import static io.strimzi.test.k8s.KubeClusterResource.cmdKubeClient;
+
+public class SetupJaeger {
```

Comment:
I think adding some JavaDoc would not hurt this class :)

Reply:
Done